eng/fv/appendix-fv.rst

```diff
@@ -0,0 +1,1318 @@
+.. SPDX-License-Identifier: CC-BY-SA-4.0
+
+.. Copyright (C) 2022 Trinity College Dublin
+
+Appendix: RTEMS Formal Model Guide
+**********************************
+
+Here we describe the various formal models of RTEMS that are currently in
+existence. The following were delivered as part of the ESA-sponsored activity,
```

Comment:
I think there should be a consistent way of acknowledging suport, but at any rate this seems out of place.

Reply:
Ok, I'll remove it.